### PR TITLE
Fix: Fix issues around incorrect tags not being sent to Grafana + Add ruff format to repo

### PR DESCRIPTION
**Pull request by jmfiola**
This fixes the tag propogation issues we were seeing in [issue 40](https://github.com/alteryx/locust-grasshopper/issues/40), and also introduces `ruff format --check .` into the quality checks pipeline, which is the request from [issue 41](https://github.com/alteryx/locust-grasshopper/issues/41).

### CODE REVIEW

**Comment by devashish2203** [comment]
Thanks @jmfiola for this and the addition of ruff format. LGTM to me merge. But will have to wait for @alteryx-sezell since she is the only code owner currently.

**Review by devashish2203 — APPROVED** [review]

**Comment by devashish2203** [comment]
Updated and merged #43 to add more code owners. But would still wait for another set of eyes on the changes before merging.

**Review by sistratton — APPROVED** [review]
LGTM

File: src/grasshopper/lib/util/utils.py
Comment (skezell) [inline src/grasshopper/lib/util/utils.py:28]: why are we getting rid of this line? do we not want to allow extra tags passed in this way anymore?

File: src/grasshopper/lib/util/utils.py
Comment (skezell) [inline src/grasshopper/lib/util/utils.py:46]: is this being added in someplace else? i thought we for sure would want the host included in the tags?

**Review by skezell — APPROVED** [review]
Approved, but left a few questions about removal of certain lines. take a look at comments before you merge ;)

File: src/grasshopper/lib/util/utils.py
Comment (jmfiola) [reply src/grasshopper/lib/util/utils.py:28]: it's all being passed in via self.tags now. All that logic can be done with just that local var, kinda like how its being done in that workflow run journey

File: src/grasshopper/lib/util/utils.py
Comment (jmfiola) [reply src/grasshopper/lib/util/utils.py:46]: its being done here: https://github.com/alteryx/locust-grasshopper/blob/cdaf03f25058c54a7cc7f3f7cb69074a084ca3fe/src/grasshopper/lib/journeys/base_journey.py#L97

so, no need to re-do it here! Tested and looks good.

**Pull request merged**
